src/pl/senla/lecture3/task3/product/ProductImpl.java

```diff
@@ -0,0 +1,52 @@
+package pl.senla.lecture3.task3.product;
+
+import pl.senla.lecture3.task3.part.IProductPart;
+
+public class ProductImpl implements IProduct{
+
+    private final IProductPart firstProductPart;
+    private final IProductPart secondProductPart;
+    private final IProductPart thirdProductPart;
+
+    public ProductImpl(IProductPart firstProductPart, IProductPart secondProductPart, IProductPart thirdProductPart) {
```

Comment:
Let's delete this all arguments constructor and replace it with default (empty) one